### PR TITLE
[ID-33] - home UI not in material app

### PR DESCRIPTION
**Pull request by suesitran**

### CODE REVIEW

**Review by hophongit98 — APPROVED** [review]

File: packages/nartus_ui_package/lib/widgets/date_label/date_label_view.dart [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  final VoidCallback? onTailIconPressed;
+
+  @override
+  Widget build(BuildContext context) {
+    return Card(
+      shape: const RoundedRectangleBorder(
+        borderRadius: BorderRadius.all(Radius.circular(NartusDimens.padding54)),
+      ),
+      color: Theme.of(context).backgroundColor,
+      elevation: NartusDimens.padding4,
```
Comment (DongNguyenQ) [inline packages/nartus_ui_package/lib/widgets/date_label/date_label_view.dart:42]: Should we write a specific NartusShadow/ NartusElevation for shadow/ elevation value ? And for the future, should we define a CardTheme for Card widget in ThemeData ?

File: packages/nartus_ui_package/lib/widgets/date_label/date_label_view.dart [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      child: Row(
+        mainAxisSize: MainAxisSize.min,
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: <Widget>[_leadingIcon(), _mainTitle(), _tailIcon()],
```
Comment (DongNguyenQ) [inline packages/nartus_ui_package/lib/widgets/date_label/date_label_view.dart:47]: Should we separate each helperMethod here to entirely new Widget according to [this article](https://www.youtube.com/watch?v=IOyq-eTRhvo) from Flutter team ?

Comment (suesitran) [reply packages/nartus_ui_package/lib/widgets/date_label/date_label_view.dart:47]: Good idea, there are onTap action on lead and tail icons, which comes with inkwell effect, so it's better to separate them.

I'll fix this

**Comment by suesitran** [comment]
2 stories in 1 PR, closed!

**Pull request closed without merging**